srcalc/src/main/java/gov/va/med/srcalc/security/SrcalcWebAuthnDetailsSource.java

```diff
@@ -0,0 +1,19 @@
+package gov.va.med.srcalc.security;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.security.authentication.AuthenticationDetailsSource;
+
+/**
+ * Provides {@link SrcalcWebAuthnDetails} objects with details from {@link
+ * HttpServletRequest}s.
+ */
+public class SrcalcWebAuthnDetailsSource
+    implements AuthenticationDetailsSource<HttpServletRequest, SrcalcWebAuthnDetails>
+{
+    @Override
+    public SrcalcWebAuthnDetails buildDetails(final HttpServletRequest context)
```

Comment:
This class is used on line 61 of `applicationContext.xml`, and Spring Security calls this method via the interface.